locksmith/src/controllers/userController.ts

```diff
@@ -0,0 +1,40 @@
+import UserOperations = require('../operations/userOperations')
+import { Request, Response } from 'express-serve-static-core'
+
+class UserController {
```

Comment:
Same here. It looks like we are not using classes for the other controllers. Any reason for the change?